src/components/ChatRoomHeader.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from 'styled-components';
+import { FiChevronLeft } from 'react-icons/fi';
+import { IChatRoomHeaderProps } from './Interface';
+
+const ChatRoomHeader = ({
+  currentUserId,
+  setCurrentUserId,
+}: IChatRoomHeaderProps) => {
+  const handleToggleUser = () => {
```

Comment:
event handle 함수는 다음과 같이 'handle + 명사 + 동사'로 작성하는것이 좋을것 같습니다.
또한 User가 아닌 CurrentUser로 작성하는것이 다른 코드와 통일성을 고려하면 가독성에 도움이 될것 같습니다.

```suggestion
  const handleCurrentUserToggle = () => {
```